src/vs/editor/contrib/find/findWidget.ts

```diff
@@ -1431,6 +1431,17 @@ registerThemingParticipant((theme, collector) => {
 		}
 	}
 
+	// Action bars
+	const toolbarHoverBackgroundColor = theme.getColor(toolbarHoverBackground);
+	if (toolbarHoverBackgroundColor) {
+		collector.addRule(`
+		.monaco-editor .find-widget .button:not(.disabled):hover,
+		.monaco-editor .find-widget .codicon-find-selection:hover {
```

Comment:
@rebornix i don't know if there's a better way to scope this line to the find in selection button since it's a button toggle (this is the only css selector I could find). let me know if this is fine or if there are better alternatives.